src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -890,6 +895,18 @@ prepareForBabelfishDatabaseDump(Archive *fout, SimpleStringList *schema_include_
 	for (i = 0; i < ntups; i++)
 		simple_oid_list_append(&catalog_table_include_oids, atooid(PQgetvalue(res, i, 0)));
 
+	PQclear(res);
+	resetPQExpBuffer(query);
+	
+	/*
+	 * Find out initialize user of current Babelfish database.
+	 */
+	appendPQExpBufferStr(query, "SELECT r.rolname FROM pg_roles r "
+						 "INNER JOIN pg_database d ON r.oid = d.datdba "
+						 "WHERE d.datname = current_database()");
+	res = ExecuteSqlQueryForSingleRow(fout, query->data);
+	babel_init_user = pstrdup(PQgetvalue(res, 0, 0));
```

Comment:
Can we reuse getBabelfishInitUser() here?

Reply:
Actually that function is defined in pg_dumpall, which has separate code base from pg_dump as both are standalone binaries.